src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -2978,7 +2977,7 @@ String getApiTailUrl(String tail) {
         if (tail.length() > 0 && !tail.startsWith("/")) {
             tail = '/' + tail;
         }
-        return "/repos/" + getOwnerName() + "/" + name + tail;
+        return "/repos/" + full_name + tail;
```

Comment:
FWIW, the owner is not provided either in the installation token, the only info we have is the full name.